generators/entity-server/templates/src/main/java/package/common/utils/is_field_with_content_type.ejs

```diff
@@ -0,0 +1,3 @@
+<% isFieldWithContentType = function(field) {
```

Comment:
You should put this function at generator-base.

Reply:
Thanks, I didn't know where to put it and I wanted to ask before asking for review! :+1: 
I just pushed the changes

Reply:
I'll squash the commits tomorrow